### ISSUE TITLE
found cookies for unexpected domain: .mbank.pl (!= online.mbank.pl)

### ISSUE THREAD

**Issue by szepczynski** [jwilk/mbank-cli#95]
Hi,

Today I started getting an error "found cookies for unexpected domain: .mbank.pl (!= online.mbank.pl)".

In cookie file I see one cookie with ".mbank.pl" domain:

```
Set-Cookie3: dtCookie=v_4_srv_2_sn_3BA3F7FB9D4B35017C3C8F2A2B462AEC_perc_100000_ol_0_mul_1_app-3Ac37c6732d88cb224_0; path="/"; domain=.mbank.pl; path_spec; discard; version=0
```

**Comment by jwilk** [comment]
Interesting. I can't reproduce it here.

Anyway, I think I'll just remove this check. I can't remember why I added it back in 2014 (807330daf239edb26858a352284f8da903764422), but it doesn't seem useful.

**Comment by szepczynski** [comment]
Maybe some A/B Testing on mbank site?

**Comment by jwilk** [comment]
Fixed in [2.2.4](https://github.com/jwilk/mbank-cli/releases/tag/2.2.4).
